python/tvm/contrib/image.py

```diff
@@ -0,0 +1,30 @@
+"""Common system utilities"""
+from __future__ import absolute_import as _abs
+import math
+import numpy as np
+
+def bilinear_weights(height, width, new_h, new_w, align_corners=False):
+    """ Helper function to generate weights for bilinear scaling """
+
+    if align_corners:
+        x_ratio = width/new_w
+        y_ratio = height/new_h
+    else:
+        x_ratio = (width-1)/(new_w-1)
+        y_ratio = (height-1)/(new_h-1)
+
+    def _bilinear_interpolation(y, x):
```

Comment:
These need to be computed inline inside the topi function by argument, instead of generating a weight matrix for all of axis. The current implementation is not efficient

Reply:
Now, there is a challenge with floor operator for this.

I would enhance it once we have floor support.